src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
My thinking is, no a transaction cannot arrive every 300ms, they all arrive at the same time the ledger closes, because we are looking only at validated ones which require the ledger to be validated.  That seems to be what is happening when I a following an orderbook, if anyone knows differently please say so.